packages/core/src/amazonqFeatureDev/userFacingText.ts

```diff
@@ -21,3 +21,9 @@ export const updateCode = 'Code has been updated. Would you like to work on anot
 export const sessionClosed = 'Your session is now closed.'
 export const newTaskChanges = 'What change would you like to make?'
 export const uploadCodeError = `Amazon Q is unable to upload workspace artifacts to S3 for feature development. For more information, see the [Amazon Q documentation](${manageAccessGuideURL}) or contact your network or organization administrator.`
+
+// Utils for logging and showing customer facing conversation id text
+export const conversationIdChat = (conversationId?: string) =>
```

Comment:
I'm trying to think if theres a slightly better way we can name these functions. Maybe `conversationIdChat -> conversationIdLog` (since its just logging the conversation id) and then `conversationIdLog -> featureDevLog` WDYT?

Reply:
mmh, it is a good question, to me it make sense because they point to the exact place they will be loged, one in the log the other one in the chat. Maybe something like: `conversationIdToChatMessage` and `conversationIdToToolkitLogs`

Reply:
The hard part for me reading it is i wouldn't exactly when to use them. It didn't click for me that one was in the chat message and one was in the output log
maybe:
`logWithConversationId` and `messageWithConversationId` ?

Reply:
That would work, sure, I will update.